### ISSUE TITLE
Reporting plugins are not mixed into the project properly

### ISSUE THREAD

**Issue by michaellasmanis** [repaint-io/maven-tiles#79]
When trying to use tiles to mixin reporting plugins into a project, only the project info reports are rendered.

In TilesMavenLifecycleParticipant:copyModel(), it appears that the reporting structure is not being copied to the new project parent.

In the attached test case [sitereporting-tiletest.tar.gz](https://github.com/repaint-io/maven-tiles/files/2147360/sitereporting-tiletest.tar.gz), the GitHub report should be generated in addition to the Project Info reports but it is not.

**Comment by michaellasmanis** [comment]
Patch submitted via Gerrit https://review.gerrithub.io/c/repaint-io/maven-tiles/+/417194

**Comment by talios** [comment]
@michaellasmanis When I run `mvn site:site` against your sitereporting project I get:

```
Execution default-cli of goal org.apache.maven.plugins:maven-site-plugin:3.3:site failed:
 A required class was missing while executing org.apache.maven.plugins:maven-site-plugin:3.3:site: 
 org/apache/maven/doxia/siterenderer/DocumentContent
```
Maybe submit a new patchset in gerrit with that test included under `src/it` so that it runs as part of the invoking `mvn clean install -Prun-its`.

**Comment by michaellasmanis** [comment]
@talios thats very odd.  the tile calls for maven-site-plugin:3.7.1 but you are getting 3.3 pulled in.  what version of maven are you running? 

I'm seeing the proper behavior with maven 3.5.3.  I cleared out all the doxia entries in ~/.m2/repository/org/apache/maven/doxia and maven fetched all the dependencies it needed.  Even setting version 3.3 for maven-site-plugin pulled in all the right stuff.

I'll work up an IT here in a bit.

**Comment by talios** [comment]
@michaellasmanis strange - will look into that.  Running 3.5.4 here.

**Comment by michaellasmanis** [comment]
I'm working on the IT now, it will be ready for your shortly.

**Comment by michaellasmanis** [comment]
@talios I think I figured out the ClassNotFound exception; seems to be related to dependency:go-offline and maven not resolving the doxia dependencies of maven-site-plugin when building the tile of the test case.

**Comment by michaellasmanis** [comment]
@talios So the ClassNotFound issue was due to the pom for the tile inheriting the default maven-site-plugin (v3.3, I'm assuming from the super-pom).  Adding the maven-site-plugin and maven-plugin-info-reports to the tile pom, gave the offline dependency walker the information it needed to pull everything down.  

I've updated the patch to include an integration test which checks for both the configuration log message as well as the generation of the GitHub issue report (this is the one that was not being generated before).

Let me know what you think.
